contentcuration/contentcuration/viewsets/file.py

```diff
@@ -61,8 +65,26 @@ def update(self, instance, validated_data):
                 ResourceSizeCache.reset_modified_for_file(instance)
 
         results = super(FileSerializer, self).update(instance, validated_data)
+        results.on_update()  # Make sure contentnode.content_id is unique
+
+        if results.contentnode:
+            results.contentnode.refresh_from_db()
+            if not len(results.contentnode.mark_complete()):
+                results.contentnode.save()
+                Change.create_change(
+                    generate_update_event(
+                        results.contentnode.id,
+                        CONTENTNODE,
+                        {"complete": True},
+                        channel_id=results.contentnode.get_channel_id(),
+                    ),
```

Comment:
Just basically copied this from the content node serializer _ensure_complete method and tweaked it to fit in here. Is there anything I've missed here that should be done/handled differently?